pkg/mkpersist/mkpersist.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
```

Comment:
This will have to absorb the functionality we're currently having here: https://github.com/lf-edge/eve/blob/master/images/rootfs-xen.yml.in#L26

And be a bit more generalized.

I think it may also benefit from a bit of a clean up.

Reply:
I have added code in new diff to take care mounting /dev/sda4 (CONFIG) onto /var/config of host file system. Apart from that I saw mountie creates symlinks in /dev/disk/by-uuid directory from UUID of partition to device. I added code to take care of this symlinking part also.
I didn't understand the clean up part completely. Can you please elaborate?